lambdas/data-migration2/src/executions.ts

```diff
@@ -21,8 +21,6 @@ const logger = new Logger({ sender: '@cumulus/data-migration/executions' });
  *   Source record from DynamoDB
  * @param {Knex} knex - Knex client for writing to RDS database
  * @returns {Promise<number>} - Cumulus ID for record
- * @throws {RecordAlreadyMigrated}
```

Comment:
It still does throw this error?

Reply:
Oops, yeah it still throws it. Let me add that back